docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: '3'
+
+services: 
+  db:
+    image: postgres
+    volumes:
+      - "data:/var/lib/postgresql/data/pgdata"

```

Don’t think you need the host side volume here - without it docker will just use an “internal” volume to persist the data. Unless you wanted to be able to access the psql data host-side for some reason. 

```suggestion
      - "/var/lib/postgresql/data/pgdata"
```

---

I wanted to be able to persist volume. And for me named volume looks better in this scope.